extensions/binding/pom.xml

```diff
@@ -34,6 +34,8 @@
     <module>org.eclipse.smarthome.binding.lirc</module>
     <module>org.eclipse.smarthome.binding.mqtt</module>
     <module>org.eclipse.smarthome.binding.mqtt.test</module>
+    <module>org.eclipse.smarthome.binding.onewire</module>
```

Comment:
In my personal alphabet, "o" comes after "n" ;)

Reply:
oops